src/js/views/forms/form/form_views.js

```diff
@@ -154,6 +156,11 @@ const LayoutView = View.extend({
           </div>
         </div>
       </div>
+      {{#if formHasWidgets}}
```

Comment:
let's make this `hasFormWidgets` where `has` `is` etc are the prefix for the check